ChopChop/ViewModels/Ingredients/IngredientBatchFormViewModel.swift

```diff
@@ -0,0 +1,68 @@
+import SwiftUI
+
+class IngredientBatchFormViewModel: ObservableObject {
+    let batch: IngredientBatch
+    var ingredient: Ingredient
+    let isEdit: Bool
+
+    @Published var selectedUnit: String
+    @Published var inputQuantity: String
+    @Published var expiryDateEnabled: Bool
```

Comment:
Why are there some ingredients which you do not allow to attach expiry dates to?

Reply:
This form is for ingredient batches and its possible for a batch to not have an expiry date